documentation.md

```diff
@@ -145,3 +145,627 @@ organization = github.Organization(organization_name="Clueless-Community")
 | `.repository_details()`     | Returns the list of repositories with their details.            |
 | `.pinned_repository()`      | Returns the list of pinned repositories with their details.     |
 | `.get_organization_links()` | Returns a dictionary of important website links of a community. |
+
```

Comment:
Do not add in the `documentation.md` file, it is for pip package documentation. A new file named `dev-documentation` is added, add your docs in that file only.

Reply:
I've added my docs in dev-documentation file and removed it from documentation.md.

Reply:
Still, it is showing 607+ lines added. I think you haven't pull the latest changes.
Only GitHub module is present in the `documentation. mf` file.
Check once again.

Reply:
I took the latest pull and updated the 'documentation.md".